torat_server/client_ishell.go

```diff
@@ -236,21 +241,20 @@ func (client *activeClient) Reconnect(c *ishell.Context) {
 	c.Stop()
 }
 
-// Remove remote file
-// TODO: Is this used? Where?
+// Shred a remote file
 func (client *activeClient) Shred(c *ishell.Context) {
-	args := shared.Shred{
-		Conf: shred.Conf{
-			Times:  3,
-			Zeros:  true,
-			Remove: true,
-		},
-		Path: strings.Join(c.Args, " "),
-	}
-	var r error
-	if err := client.RPC.Call("API.Shred", &args, &r); err != nil {
-		c.Println(red("[!] Could not shred path:", args.Path, err, r))
+	var s shared.Shred
+	s.Path = strings.Join(c.Args, " ")
+	s.Times = 3
+	s.Zeros = true
+	s.Remove = true
+
+	if err := client.RPC.Call("API.Shred", &s, &void); err != nil {
+		c.Println(red("[!] Could not shred path: ", s.Path))
+		c.Println(red("[!] ", err))
```

Comment:
c.Println(red("[!] Could not shred path", s.Path, err))

Reply:
Why stack the error in a single line? This is easier to read

Reply:
Mostly by convention

Reply:
I do this elsewhere for readability's sake. Do we want to change that too? Would like to keep consistent.

Reply:
Yes we want to change that too probably.

Reply:
okay maybe we just patch this one then I send another PR with the full fix for all the files?

Reply:
sure